src/rulesReport.ts

```diff
@@ -68,13 +70,15 @@ export const reportRules = (groupBy: GroupBy) => {
   processOffenses(reportSimpleComputed)
   processOffenses(reportComponentFiles)
```

Comment:
Can we keep alphabetic order here?

Reply:
you mean per section? like this:

processOffenses(reportComponentFilenameCasing)
processOffenses(reportComponentFiles)
processOffenses(reportDirectiveShorthands)
processOffenses(reportFullWordComponentName)
processOffenses(reportMultiAttributeElements)
processOffenses(reportPropNameCasing)
processOffenses(reportQuotedAttributeValues)
processOffenses(reportSelfClosingComponents)
processOffenses(reportSimpleComputed)
processOffenses(reportTemplateSimpleExpression)

Reply:
fix in last commit